docs/api/expect.md

```diff
@@ -5,6 +5,22 @@ title: Expect
 
 When you're writing tests, you often need to check that values meet certain conditions. `expect` gives you access to a number of "matchers" that let you validate different things on the `browser` or an `element` object.
 
+## Default Options
+
+```js
+{
+    wait: 2000, // ms to wait for expectation to succeed
+    interval: 100, // interval between attempts
+}
+```
+
+Set options like this:
+```js
+beforeSession () { // before hook works as well
+    require('expect-webdriverio').setOptions({ wait: 5000 })
```

Comment:
Even though this would work the actual way of modifying these defaults is by setting the `waitforTimeout` and `waitforInterval` options in your config. So only if you want to use specific timeouts for the assertions you should do this.

Can we add a sentence that mentions that these defaults are connected to the `waitforTimeout` and `waitforInterval` options?

Reply:
Thanks I've updated this, let me know if it's ok.